### ISSUE TITLE
SyncTrayzor can't start Syncthing

### ISSUE THREAD

**Issue by WhiteSp1rit** [canton7/SyncTrayzor#800]
SyncTrayzor portable x64 version 1.1.29
Syncthing v1.27.0
OS: Windows 7 SP1 Ultimate

Syncthing by itself when started directly works with no issues.
SyncTrayzor launches, but when I click "Start Syncthing", it will fail to launch it, with "Syncthing stopped" status.

Logs:

SyncTrayzor:

```
2024-02-18 12:19:35.5556 #9104 [Info] SyncTrayzor.Syncthing.SyncthingProcessRunner: Starting syncthing: somefilepath\SyncTrayzorPortable-x64\data\syncthing.exe 
2024-02-18 12:19:35.6025 #9104 [Info] SyncTrayzor.Syncthing.SyncthingProcessRunner: Syncthing exited. Restarting... 
2024-02-18 12:19:35.6025 #9104 [Info] SyncTrayzor.Syncthing.SyncthingProcessRunner: Starting syncthing: somefilepath\SyncTrayzorPortable-x64\data\syncthing.exe 
2024-02-18 12:19:35.6337 #9104 [Info] SyncTrayzor.Syncthing.SyncthingProcessRunner: Syncthing exited. Restarting... 
2024-02-18 12:19:35.6337 #9104 [Info] SyncTrayzor.Syncthing.SyncthingProcessRunner: Starting syncthing: somefilepath\SyncTrayzorPortable-x64\data\syncthing.exe 
2024-02-18 12:19:35.6572 #9104 [Info] SyncTrayzor.Syncthing.SyncthingProcessRunner: Syncthing exited. Restarting... 
2024-02-18 12:19:35.6572 #9104 [Info] SyncTrayzor.Syncthing.SyncthingProcessRunner: Starting syncthing: somefilepath\SyncTrayzorPortable-x64\data\syncthing.exe 
2024-02-18 12:19:35.6855 #9104 [Warn] SyncTrayzor.Syncthing.SyncthingProcessRunner: 4 restarts in less than 00:01:00: not restarting again
``` 

Syncthing:

```
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0
runtime.asmstdcall()
	runtime/sys_windows_amd64.s:65 +0x75 fp=0x22fca0 sp=0x22fc80 pc=0x473eb5
rax     0x0
rbx     0x1a43f00
rcx     0x1aa4240
rdi     0x7fffffde000
rsi     0x22fea0
rbp     0x22fde0
rsp     0x22fc78
r8      0x0
r9      0x22fee0
r10     0x1a75a18
r11     0x21
r12     0x22fec0
r13     0x1
r14     0x1a43360
r15     0x0
rip     0x0
rflags  0x10293
cs      0x33
fs      0x53
gs      0x2b
```
this is repeated on each start attempt

**Comment by tomasz1986** [comment]
Syncthing is no longer compatible with Windows 7. The last version that works is Syncthing v1.27.0. If you want to continue using it, you will need to manually downgrade by overwriting the `syncthing.exe` binary, and then disable automatic upgrades.

**Comment by WhiteSp1rit** [comment]
> Syncthing is no longer compatible with Windows 7. The last version that works is Syncthing v1.27.0. If you want to continue using it, you will need to manually downgrade by overwriting the `syncthing.exe` binary, and then disable automatic upgrades.

You didn't read the post then? I already wrote that Syncthing by itself runs fine with no issues. Already using v1.27.0

**Comment by tomasz1986** [comment]
The crash message is exactly the same as in https://github.com/syncthing/syncthing/issues/9282, which means that this is the same Syncthing problem being unable to run on Windows 7, not something caused by SyncTrayzor. I believe SyncTrayzor by default tries to upgrade Syncthing to the newest version immediately at launch, which then likely leads to the crash.

**Comment by WhiteSp1rit** [comment]
> The crash message is exactly the same as in [syncthing/syncthing#9282](https://github.com/syncthing/syncthing/issues/9282), which means that this is the same Syncthing problem being unable to run on Windows 7, 

re-iterating - Syncthing is running with no issues. If I start it myself. Folders sync, everything works. Only SyncTrayzor can't start it.

> I believe SyncTrayzor by default tries to upgrade Syncthing to the newest version immediately at launch, which then likely leads to the crash.

I already said that I turned off autoupdate.

**Comment by tomasz1986** [comment]
> I already said that I turned off autoupdate. Right now the version of Syncthing is the one I stated, and according to logs SyncTrayzor does not try to update it.

Please specify how exactly you have automatic upgrades. There are multiple ways to do it, and something clearly isn't working, at least in combination with SyncTrayzor.

**Comment by WhiteSp1rit** [comment]
> or possibly yet another, newer Syncthing binary present in a different location that SyncTrayzor tries to start.

Yeah that turned out to be the case, as I misunderstood which file SyncTrayzor is trying to launch.
In SyncTrayzor portable archive, there is a syncthing.exe at the root and that is what I've replaced and then launched manually.
Turns out SyncTrayzor just ignores it, and creates a new "data" folder where it launches a different copy of syncthing from. 
Very confusing behaviour. And it leaves the original syncthing.exe at the root folder, despite not using it.

After I replaced the syncthing.exe in the data folder everything is working now, so this issue can be closed.
